L78-python-rich-server-context.md

```diff
@@ -0,0 +1,73 @@
+Title
+----
+* Author(s): Chris St. Pierre
+* Approver: a11r
+* Status: Draft
+* Implemented in: python
+* Last updated: 2021-02-17
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+Expose context status code, details, and trailing metadata on the
+server side.
+
+## Background
+
+Currently, on the server-side `ServicerContext` object, status code,
+details, and trailing metadata are private. As a result, an
+interceptor cannot take action based on any of those properties.
+
+### Related Proposals
+
+N/A
+
+## Proposal
+
+We will add three methods to the servicer context objects:
```

Comment:
Can you explicitly name those interfaces? I believe there are four. There are sync and async, as well as interceptor and non-interceptor for each of those.